pkg/providers/instance/instance.go

```diff
@@ -93,7 +93,6 @@ type Provider struct {
 }
 
 func NewProvider(
-	_ context.Context, // TODO: Why is this here?
```

Comment:
Chestersons fence. Looking back in the history this seems to follow the pattern of the aws provdier, but since has changed